src/main/drivers/pwm_output.c

```diff
@@ -137,14 +137,16 @@ static void pwmWriteStandard(uint8_t index, uint16_t value)
 
 static void pwmWriteOneshot(uint8_t index, uint16_t value)
 {
-	//value = value >> 2;
-	//value = value << 2;
-    *motors[index]->ccr = (uint16_t)((float)value*1.5f);  //Expecting 8MHz timer, multiply by 1.5 to scale to 12MHz timer
+    *motors[index]->ccr = (uint16_t)((float)value);
 }
 
 static void pwmWriteMultiShot(uint8_t index, uint16_t value)
 {
-    *motors[index]->ccr = (uint16_t)((float)(value-1000) / 4.1666f)+ 60;
+#if defined(STM32F40_41xxx) || defined(STM32F411xE)
```

Comment:
would be good to use integer calculations here if possible. This is already update in latest master.